guide/05-working-with-the-spatially-enabled-dataframe/part2_data_io_reading_data.ipynb

```diff
@@ -0,0 +1,5229 @@
+{
```

Comment:
<p><code>spatial_filter</code> can be used to query the results by using a spatial relationship with another geometry. The spatial filtering is even more powerful when integrated with <a href="https://developers.arcgis.com/python/guide/part1-introduction-to-geoenrichment/" target="_blank">Geoenrichment</a>. Let's use this approach to filter our results for the state of Idaho. In this example, we will: </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='96'/>